test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -44,15 +54,24 @@
             "py2": ['36810',
                     # for shipping pillow<=6.2.2 - the last available version for py2
                     '38449', '38450', '38451', '38452'],
-            "py3": ['36810']
```

Comment:
What is vulnerability 36810?

Reply:
I believe i fixed this, can you refresh your github page

Reply:
No, I mean that 36810 has been removed from the list, so just want to make sure we don't encounter it again.

I guess if the test passes, then we will know that it doesn't affect the image anymore.